openshift/README.md

```diff
@@ -84,6 +85,55 @@ as well as Jenkins slave images for [Maven](https://github.com/openshift/jenkins
 These next set of steps build upon the steps just executed above, leveraging the OpenShift Jenkins slave image for NodeJS to launch the sample
 job in a Jenkins slave provisioned as Kubernetes Pod on OpenShift.
 
+## OSX slave setup
+
+It is not possible to run OSX in a container, thus we need to connect our OpenShift cluster to an OSX system outside of OpenShift.
+In Jenkins terms, it is an "OSX agent" or "OSX slave".
+
+In order to do that, you must first configure the OSX system. This will be replaced by an Ansible script in the future.
+
+All of the following operations on OSX machine requires `sudo`.
+
+First, create a Jenkins user on your OSX machine with username `jenkins` and password `Password1`:
+
+        . /etc/rc.common
+        dscl . create /Users/jenkins
+        dscl . create /Users/jenkins RealName "Jenkins Agent"
+        dscl . passwd /Users/jenkins Password1
```

Comment:
Let's change this to use some env variable and print this on start rather than provide fixed value here. 
Knowing this would allow anyone to ssh to your machine from local network, so it's important that users would not blindly copy this etc.